source/src/microapp/cs_MicroappInterruptHandler.cpp

```diff
@@ -544,6 +549,27 @@ void MicroappInterruptHandler::onBlePeripheralNotififyDone(uint16_t connectionHa
 	MicroappController::getInstance().generateSoftInterrupt(CS_MICROAPP_SDK_TYPE_BLE, CS_MICROAPP_SDK_BLE_PERIPHERAL);
 }
 
+void MicroappInterruptHandler::onAssetAccepted(AssetAcceptedEvent& event) {
+	LogMicroappInterrupDebug("onAssetAccepted");
+	uint8_t* outputBuffer = getOutputBuffer(CS_MICROAPP_SDK_TYPE_ASSETS, CS_MICROAPP_SDK_ASSET_EVENT);
+	if (outputBuffer == nullptr) {
+		return;
+	}
+	asset_id_t assetId = event._primaryFilter.getAssetId(event._asset);
+	microapp_sdk_asset_t* asset = reinterpret_cast<microapp_sdk_asset_t*>(outputBuffer);
+	asset->header.messageType = CS_MICROAPP_SDK_TYPE_ASSETS;
+	asset->type               = CS_MICROAPP_SDK_ASSET_EVENT;
+	memcpy(asset->event.assetId, assetId.data, sizeof(asset_id_t));
```

Comment:
Asset id struct might get more fields.
```suggestion
	memcpy(asset->event.assetId, assetId.data, sizeof(assetId.data));
```